src/discord/events/message/registerTeam.ts

```diff
@@ -0,0 +1,214 @@
+import Discord from 'discord.js';
+
+import { Config } from '../../../entities/config';
+import logger from '../../../logger';
+import { colors } from '../../constants';
+import { Team } from '../../../entities/team';
+
+class TeamInfo {
```

Comment:
An interface should suffice for this data type. We can use a class but it'll just create overhead and require us to write more tests with little added value.